src/macros-by-example.md

```diff
@@ -59,6 +59,10 @@ matches, in a structure that mimics the structure of the repetition encountered
 on a successful match. The job of the transcriber is to sort that structure
 out.
 
+> **Edition Differences**: In the 2018 edition, a new Kleene operator was
```

Comment:
What `?` means should be defined outside of the edition differences section. And then the edition differences section should just state that it's not available in the 2015 edition.